src/Proto3/Wire/Decode.hs

```diff
@@ -122,8 +122,8 @@ data ParsedField = VarintField Word64
 -- key, in their reverse occurrence order.
 --
 --
-decodeWire :: B.ByteString -> Either String RawMessage
-decodeWire = decodeWire0 combineSeen' Nothing close
+decodeWireMessage :: B.ByteString -> Either String RawMessage
```

Comment:
Is the name change necessary?  If we could keep the original name here and then rename the other function then this wouldn't be a breaking change, I think

Reply:
So the original was `decodeWire :: B.ByteString -> Either String [(FieldNumber, ParsedField)]`
A previous change _erroneously_ caused it to be `decodeWire :: B.ByteString -> Either String RawMessage` in main, which is different from the signature at the last release.

Reply:
I should have communicated that in the PR message above, as it might not have been clear from the conversation

Reply:
Ah, now I understand